### PR TITLE
Fix unlimited use of potion.

### PR DESCRIPTION
**Pull request by kaninhot004**
* **Addressed Issue(s)**: None

* **Server Mode**: Renewal

* **Description of Pull Request**: Fix unlimited item use exploit.

### CODE REVIEW

**Comment by gmragnarok** [comment]
Is the correct script like this?

**Comment by kaninhot004** [comment]
Not sure but at least it fix the unlimited use bug.

**Review by aleos89 — APPROVED** [review]
The only comment I have is if officially it's supposed to show the Kyrie skill effect or not.

File: db/re/item_db_usable.yml [hunk 1]
```diff
@@ -14426,14 +14426,14 @@ Body:
...
-    Type: Delayconsume
+    Type: Usable
     Weight: 30
     Delay:
       Duration: 1000
       Status: Reuse_Limit_F
     Script: |
       percentheal 100,100;
-      itemskill "PR_KYRIE",10;
+      sc_start SC_KYRIE,100000,10;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:14436]:
```suggestion
      sc_start SC_KYRIE,100000,10;
      skilleffect "PR_KYRIE",0;
```

**Review by Atemo — COMMENTED** [review]
(the item was consumed after using the skill, not sure we can call it an exploit)

**Comment by kaninhot004** [comment]
More explaination: Use this item will heal you 100% HP and SP then cancel the cast. Repeat this forever.

**Pull request merged**